src/components/dialogs/dialogAlert/dialogAlertContent/dialogAlertContent.tsx

```diff
@@ -0,0 +1,14 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { type ComponentPropsWithoutRef } from 'react';
+
+export interface IDialogAlertContentProps extends ComponentPropsWithoutRef<'div'> {}
+
+/**
+ * `DialogAlert.Content` component.
+ */
+export const DialogAlertContent: React.FC<IDialogAlertContentProps> = ({ className, ...otherProps }) => {
+    return <div className={classNames('overflow-auto px-4 md:px-6', className)} {...otherProps} />;
```

Comment:
I saw that on Figma the vertical padding is applied to a inner div, I think that the `content` should also have a default vertical padding. What do you think?

Reply:
Per designs, only horizondal padding should be added. Vertical padding is left to the content to define.